.env.example

```
@@ -3,5 +3,5 @@ SENTRY_DSN=You_DSN
 GOVUK_NOTIFY_API_KEY=your_api_key
 DATA_MANAGEMENT_EMAIL=data_management_team_email_address
 JIRA_URL=your_jira_url
-JIRA_BASIC_AUTH=your_jira_basic_auth
+JIRA_API_KEY=your_jira_api_token
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JIRA_API_KEY key should go before the JIRA_URL key

(UnorderedKey)